src/Robo/Commands/Sync/RefreshCommand.php

```diff
@@ -54,6 +54,7 @@ protected function refreshMultisite($multisite_name) {
    * local db, re-imports config, and executes db updates.
    *
    * @command sync:refresh
+   * @executeInDrupalVm
```

Comment:
We've bounced back and forth with this. In the end, we really just need to run this from inside the VM via a `vagrant ssh`. The redispatch is problematic, which is why it was removed from BLT 9.

Reply:
Yep 👍 - but it's just inconsistent in `8.9` - I can run `$ blt setup` from the host, but not `$ blt sync:refresh` which is why I thought this was appropriate.
I'm still trying to get onto `9.0` completely, so I don't have much more need for these fixes and used this as a patch anyway. I guess it would just be if this is valuable to other folks.
Thanks for the review!